tasks/main.yml

```diff
@@ -24,6 +24,7 @@
   shell: /usr/share/elasticsearch/bin/plugin {% if elasticsearch_file_name|get_major_version == 1 %}--{% endif %}install {{ item }}
   with_items: "{{ plugins_to_install }}"
   register: plugin_response
+  when: elasticsearch_file_name|get_major_version < 5
```

Comment:
If I'm not mistaken, some plugins could be installed with ES5, right?
It's just site plugins that are no longer allowed I think.